### ISSUE TITLE
v8: Ensure all other elements can't be focused while infinite editing is active (Focus/Tab lock)

### ISSUE THREAD

**Issue by BatJan** [umbraco/Umbraco-CMS#4525]
In order to ensure accessibility and a pleasant user experience when using infinite editing I suggest that we include the inert polyfill from https://github.com/WICG/inert, which will mimic expected browser behavior once this attribute is implemented into browsers.

Whenever an element has the "inert" attribute itself and it's children can't be interacted with.

Currently when tabbing through the infinite overlay focus is not kept within it, which is demonstrated in this little video https://www.youtube.com/watch?v=es8i3ALmmm0&feature=youtu.be - Notice how the app section items are being highlighted and also the pagename in the layer behind the infinite layer.

I have been poking around the codebase this evening and I think I have an idea about how this can be achieved so I will open a PR for it - I will probably need some HQ input for some of it. But I'll let you know when I have something that can be checked out etc.

To get an idea about what I suggest the outcome should be please have a look at this CodePen demo, which illustrates the concept perfectly (Not using inert though): https://codepen.io/matuzo/pen/GrNdvK

**Comment by melvinvanrookhuizen** [comment]
@nul800sebastiaan tested against v8.9, seems solved. asl also #5463 is merged

**Comment by nul800sebastiaan** [comment]
Late reply, but yes, let's close this one as fixed! 🎉
